db/migrate/20161205203956_create_join_table_organization_project.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableOrganizationProject < ActiveRecord::Migration
+  def change
+    create_join_table :organizations, :projects do |t|
+      t.index [:organization_id, :project_id]
```

Comment:
Do you think the query planner use two single column indexes instead of the two compound indexes?